admin_guide/backup_restore.adoc

```diff
@@ -186,6 +186,11 @@ certificates in order to run pods.
 [[cluster-restore-single-member-etcd-clusters]]
 == Cluster Restore for Single-member etcd Clusters
 
+[NOTE]
+====
+The etcd v2 commands that you use in these steps are valid for both v2 and v3.
+====
+
```

Comment:
IIRC starting with 3.6 only v3 data model is used[1], so I think v2 data is not needed.

[1] https://docs.openshift.com/container-platform/3.6/install_config/upgrading/migrating_etcd.html

Reply:
@e-minguez, interesting. I know I saw some discussion on one of my etcd bugs about you needing to migrate the v3 database even if you're still using the v2 model, but I don't recall discussion of the need to move v2 data if you were using v3. 

@wanghaoran1988, what do you think? Will you please review the rest of this change?